app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ActiveRecord::Base
+  validates :body, presence: true
+  belongs_to :user
```

Comment:
Yes, I knew I was forgetting something so I wanted to post this as soon as possible.